app/lib/database/repositories/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+
+const modelsCreator = require('../models');
+const Repository = require('./Repository.js');
+
+/**
+ * Object for holding repository classes defined in files in this directory. 
+ * If repository is not defined here explicitly then it will be created implicitly via models mapping
+ * NOTE:
+ *  1. Instances are created here, so metioned files should export classes not instances.
+ *  2. The object have to keep each repository under key the same as corresponding model is kept. 
+ */
+const specificallyDefinedRepositories = {};
+
+module.exports = (sequelize) => {
+    const modelName2Repository = Object.entries(modelsCreator(sequelize)).map(([modelName, model]) => {
```

Comment:
So in database manager models could be acquired from repositories, but not sure if it is needed, because rather, only static methods of models are in use

Reply:
What do you mean by models could be acquired from repositories?

Sequelize is active record oriented, which is in my opinion a really bad anti-pattern because it completely break single responsibility principle. This is why in bkp the choice has been made to create repositories (data mapper pattern), and to keep this useful we need to never use persistence-related functions from model (such as `save` for example`). Hence, models should be converted to entities as soon as possible, and manipulated as little as possible (never outside of repositories for persistence and low level services as data representation).

Reply:
ok, understand now, so models will be removed from DatabaseManager